### PR TITLE
Add half-and-half suru strip to IoT Appstore page

### PR DESCRIPTION
**Pull request by MeltySnow**
## Done

Added half-and-half Suru strip to IoT digital signage page.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things/appstore
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure that the design matches the [specification](https://github.com/canonical-web-and-design/web-squad/issues/1463)

## Issue / Card

Fixes canonical-web-and-design/web-squad#1463

## Screenshots
image

### CODE REVIEW

**Comment by pmahnke** [comment]
@anasereijo perhaps on mobile, you should hide the background?

image

**Comment by anasereijo** [comment]
Hey @MeltySnow, as discussed please amend the following:

- [ ] replace snap store logo with this [one](https://assets.ubuntu.com/v1/c3b8a9dd-Snap_Store_shopping_icon.svg) 

- [ ] `background-size: 46.2% 68.6%, 38.4% 100%, 16.8% 100%, 40.5% 100%;`

- [ ] mobile: background make it the gradient

**Comment by MeltySnow** [comment]
@anasereijo I've pushed a change that should address these issues.

File: templates/internet-of-things/appstore.html [hunk 1]
```diff
@@ -8,27 +8,45 @@
...
+          <li class="p-inline-list__item">
+            <a class="p-button--positive" href="https://www.brighttalk.com/webcast/6793/290087" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Webinar', 'eventLabel' : 'Watch the Path to IoT revenue webinar', 'eventValue' : undefined });"><span class="p-link--external">Watch the 'Path to IoT revenue' webinar</span></a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="/internet-of-things/contact-us?product=appstore" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us', 'eventLabel' : 'Telcos - Top CTA' });" class="p-button--neutral js-invoke-modal">Contact us</a>
+          </li>
+        </ul>
+      </div>
+      <div class="col-6 u-align--center u-vertically-center u-hide--small">
+        <img src="https://assets.ubuntu.com/v1/c3b8a9dd-Snap_Store_shopping_icon.svg" width="250" alt="">
```
Comment (solazio) [inline templates/internet-of-things/appstore.html]: use `{{ ASSET_SERVER_URL }}` instead of `https://assets.ubuntu.com/v1/`

**Comment by MeltySnow** [comment]
I've updated the image URL.

**Comment by solazio** [comment]
@MeltySnow there are some conflicts now. Please fix.

**Pull request merged**